blocks/modal/modal.css

```diff
@@ -0,0 +1,49 @@
+body.modal-open {
+  overflow: hidden;
+}
+
+.modal dialog {
+  overscroll-behavior: none;
+  border: 1px solid #ccc;
+  width: 100vw;
+}
+
+.modal dialog .modal-content {
+  overflow-y: auto;
+  overscroll-behavior: none;
+  max-height: calc(100dvh - 60px);
+}
+
+@media (width >= 600px) {
```

Comment:
Correct the breakpoints, see style.css for breakpoints

Reply:
This is addressed, corrected the breakpoints to 1280px